snap/implicit.go

```diff
@@ -48,6 +48,8 @@ var implicitSlots = []string{
 	"network-observe",
 	"network-setup-observe",
 	"opengl",
+	"openvswitch",
```

Comment:
Based on the fact that openvswitch is an implicit slot, doesn't exist in the core snap and /run/openvswitch/db.sock therefore will never exist on all snaps, it should be added to implicitClassicSlots. Contrast that with openvswitch-support which should be in implicitSlots since all it does is make sure a module is loaded and that module might be provided in a kernel snap on all snaps.